### DISCUSSION TITLE
Approved since May 26 but still no benefits?

### DISCUSSION THREAD

**Discussion by j1n-o9r** [community/community#56623] [GitHub Education]
### Hi everyone,

Hello,

I guess the title is self explaining....

After da long verfication process I finally got appoved for GitHub Education since Im currently studying CS. But because of form issues (profile bs, which I had to google) I got rejected multiple times, before passing the verification process. 

Now Im very insecure about the process was enden correctly. 

Unfortantely I got no Mail that proves the application, I only recieved four mails which proved the rejections before. Since it all was in a short period of time, it´s possible there occured an error at our school mail services. (We got hacked last year, and are still recovering from massive damage in our the internet infrastructure of our school) 

So here is my question: 

Is it possible that the proper process of the education activation takes this long or should I have confirmed the last step of the verification via mail as well? 

Thanks in advance!

![githubproble1](https://github.com/community/community/assets/52105660/e5bac291-8891-41bb-8b05-ca83f9be40d9)
![githubproble2](https://github.com/community/community/assets/52105660/141e8a95-f785-4479-a842-c28f216d9b1a)

**Comment by Ntshangase** [comment]
Hi @lptrm 

It looks like your process ended properly. 

You should just keep checking your profile and expect to get verified in the next 72hours then you can start enjoying your student benefits.

**Reply by sabbir-rahman-taif** [reply]
My request is also approved But I didn't get the benefits yet. @Ntshangase 
![github](https://github.com/community/community/assets/69286151/931639e0-bf20-4c1d-8743-32b68c1e01e8)

**Comment by samus-aran** [comment]
We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
